satpy/tests/reader_tests/test_seviri_l1b_native.py

```diff
@@ -584,7 +601,7 @@ def create_test_header(earth_model, dataset_id, is_full_disk, is_rapid_scan):
                     reference_grid: {
                         'ColumnDirGridStep': column_dir_grid_step,
                         'LineDirGridStep': line_dir_grid_step,
-                        'GridOrigin': 2,  # south-east corner
+                        'GridOrigin': gridorigin,  # south-east corner
```

Comment:
This comment is now outdated. You could explain in the docstring that the default value of 2 represents the south-east corner and then remove this comment.